maven/build-scan/publish/action.yml

```diff
@@ -116,10 +131,44 @@ runs:
           nbScans=$(find ${{ env.BUILD_SCAN_DIR }}${version} -type f -name "scan.scan"  | wc -l)
           for ((i=1; i <= $nbScans; i++))
           do
-            echo "Publishing Build Scan ${i}/${nbScans}"
+            echo "Publishing Build Scan (version ${version}) ${i}/${nbScans}"
             
             # publish build scan
-            mvn gradle-enterprise:build-scan-publish-previous
+            mvn gradle-enterprise:build-scan-publish-previous | tee build.out
+
+            # collect build scan link
+            scanLink=$(grep -A1 "Publishing build scan..." build.out  | tail -n 1  | sed 's/\[INFO\] //')
+            if [[ ! -z "$scanLink" ]]
+            then
+              scanLinks="${scanLinks},[Link $((publishedScans++))]($scanLink)"
```

Comment:
I think these links would be more useful if they were raw so the user could see the URL. If we have a nice human-readable text for the build then that would be good, but the "Link 1" text just obfuscates things.

Reply:
You're totally right 👍